images/installer/Dockerfile.upi.ci

```diff
@@ -88,11 +88,14 @@ RUN mkdir /output && HOME=/output && \
     ibmcloud plugin install cloud-internet-services -f && \
     cp /usr/local/bin/ibmcloud /bin/ibmcloud && \
     rm -f $HOME/.curlrc && \
+    chmod -R g=u "$HOME/.bluemix/" && \
     ibmcloud version && \
-    ibmcloud plugin list
+    ibmcloud plugin list && \
+    ls -la /output/.bluemix && \
+    ls -la /output/.bluemix/plugins/
 
 RUN chown -R 1000:1000 /output
```

Comment:
Pls change this line to `RUN chown 1000:1000 /output`

Reply:
@MayXuQQ In https://github.com/openshift/installer/pull/6624/files, we changed the line to `RUN chown -R`, because we thought uid in container pod will be `1000`, after that, we found the uid was not `1000`, we have to use `chmod -R g=u "$HOME/.bluemix/"` to resolve it, so I suggested to change the line back to `RUN chown 1000:1000 /output`. Or else, do you mean `chmod -R g=u "$HOME/.bluemix/"` will resolve the permission issue, also need `RUN chown -R 1000:1000 /output` ?